h/static/scripts/app.coffee

```diff
@@ -110,6 +110,7 @@ module.exports = angular.module('h', [
 .directive('spinner', require('./directive/spinner'))
 .directive('tabbable', require('./directive/tabbable'))
 .directive('tabReveal', require('./directive/tab-reveal'))
+.directive('viaLinkDialog', require('./directive/via-dialog'))
```

Comment:
@RawKStar77: This mentions a via-dialog file which is missing from this commit. Can you provide it?